tests/manage/storageclass/test_multiple_sc_comp_rep_data_delete.py

```diff
@@ -0,0 +1,101 @@
+import logging
+
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.resources.pod import get_fio_rw_iops
+from ocs_ci.framework.testlib import ManageTest, tier1
+from ocs_ci.framework.pytest_customization.marks import (
+    skipif_external_mode,
+    skipif_ocs_version,
+)
+from ocs_ci.ocs.exceptions import (
+    PoolDataNotErased,
+    PvcNotDeleted,
+)
+from ocs_ci.ocs.cluster import get_byte_used_by_pool
+from ocs_ci.ocs.resources.pod import delete_pods
+from ocs_ci.ocs.resources.pvc import (
+    delete_pvcs,
+    get_all_pvcs_in_storageclass,
+)
+from time import sleep
```

Comment:
this should be above the ocs_ci import statements, at line 2

Reply:
Done